docs/T1K_README.md

```diff
@@ -1,27 +1,47 @@
 # T1K Genotyper
-["The ONE genotyper for Kir and HLA](https://github.com/mourisl/T1K/).
-Currently implemented as a tool, the [usage guide](https://github.com/mourisl/T1K/tree/v1.0.2#usage) explains all the possible tool options. Currently, the cwltool is limited, but will be expanded with future development
-
-## tools/t1k.cwl
-### Required Inputs
- Input reads:
- - `reads`: read1 fastq reads if paired-end (omit if single end or interleaved)
- - `mates`: mates (read2) fastq reads if paired-end (omit if single end or interleaved)
- - `single_end`: fastq reads if single-end (omit if paired-end or interleaved)
- - `interleaved`: fastq reads if interleaved (omit if single end, or read1 and mates are separate files)
- - `bam`: if bam input (use if input reads are bam and not fastq)
- Reference:
+[The ONE genotyper for Kir and HLA](https://github.com/mourisl/T1K/).
+Currently implemented as a tool, the [usage guide](https://github.com/mourisl/T1K/tree/v1.0.5#usage) explains all the possible tool options.
+
+## Inputs
+### Reference
  - `t1k_ref_tar`: Tar ball with T1K reference
  - `t1k_ref_path`: String of path to un-tarred hla ref, i.e. kiridx/kiridx_rna_seq.fa
  - `output_basename`: Prefix string for output file names. Default inferred from input
-### Optional resource
+
+ ### Input Reads
+
+The reads for T1K can be delivered as either BAM or fastq. The [T1K documentation](https://github.com/mourisl/T1K/tree/v1.0.5#inputoutput) outlines the various ways in which these files can be delivered. These match this tool's inputs: 
+
+ - `reads`: Port for `-1` input, read1 fastq reads if paired-end (omit if single end or interleaved)
+ - `mates`: Port for `-2` input, mates (read2) fastq reads if paired-end (omit if single end or interleaved)
+ - `single_end`: Port for `-u` input, fastq reads if single-end (omit if paired-end or interleaved)
+ - `interleaved`: Port for `-i` input, fastq reads if interleaved (omit if single end, or read1 and mates are separate files)
+ - `bam`: Port for `-b` input, if bam input (use if input reads are bam and not fastq)
+
+### Presets
+
+T1K can be used for RNA-seq, WES, or WGS analysis. The default parameters are for RNA-seq. If you wish to perform a different analysis, you can change the `preset` field. That field has the following options:
```

Comment:
You say the defaults are for RNAseq, but I am guessing it's also for WXS given there is a specific WGS preset